spec/models/original_record_spec.rb

```diff
@@ -147,9 +147,10 @@
 
     it 'reloads content_type' do
```

Comment:
Since it's not being saved after line 151, can we not assume that subject.content_type is going to be the same after reload?